clang/test/Sema/attr-malloc_span.c

```diff
@@ -0,0 +1,83 @@
+// RUN: %clang_cc1 -verify -fsyntax-only %s
+// RUN: %clang_cc1 -emit-llvm -o %t %s
```

Comment:
Why the additional `-emit-llvm` case?
I know the attr-malloc.c test has it, but we're not testing CodeGen in Sema tests.